modules/bmo-configuring-ignition-userdata.adoc

```diff
@@ -0,0 +1,50 @@
+// This module is included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bmo-configuring-ignition-userdata_{context}"]
+= Configuring Ignition-based userData for Fedora CoreOS
+
+The Fedora CoreOS image requires the `userData` field to use an Ignition configuration instead of a `cloud-init` script. Ignition is the utility Fedora CoreOS uses for initial system configuration.
```

Comment:
maybe let's add a reference to upstream doco: https://docs.fedoraproject.org/en-US/fedora-coreos/producing-ign/

technically Ignition is created from a butane file with butane CLI but I feel like that's too detailed for BMaaS document (this is not a butane/ignition manual after all) so I think pasting final Ignition below is good (that's all what I did in my testing anyway). But if we have doco link, if people want to know more they can read up on it.

Reply:
Done.